### PR TITLE
Data: Updater: Pull Entity/Content Data

### PR DESCRIPTION
**Pull request by gblakeman**
Changes the `run-data-update.mjs` module to accommodate “Content” and “Entity” data types, in addition to the 1.0 “Base” types ([schema reference](https://www.figma.com/file/0bzWAwvkVmb4JJ5l0sBKcZ/Realish-2.0-%E2%80%93-Persona-Airtable-Schema)).

This new version of the script will download all table data and process Airtable’s record linking IDs to create local data files in a schema that won’t need much (if any) additional processing and cross-referencing.

To try it: `npm run data-update:beta` (new stuff is only available in beta*). You will need to generate your Airtable API key and have access to the base.

----

### Additional Notes
* Also adds the ability to call beta data (`npm run data-update:beta` version production: `npm run data-update`). Draft data is still ignored.
* Images are currently ignored and handled in their 1.0 fashion. This will come in a later followup.
* Added a bunch of comments, but still foregoing proper docs until the dust settles.
* Updated the [README](https://github.com/linkedinlabs/realish-figma/tree/v2/entity-data#managing-data-optional).
* Added the `RealishAssignmentObject` type.
* Adds some new “Base” data types: gender, job types, and pronouns to `Data`.
* Allows `Presenter` to flag certain data as hidden (`visible: false`) to hide in the UI.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +34,8 @@
     "lint:css": "npm run lint:stylelint",
     "lint:staged": "lint-staged",
     "lint:svelte": "svelte-check",
-    "data-update": "node -r dotenv/config src/data/run-data-update.mjs dotenv_config_path=.env"
+    "data-update": "node -r dotenv/config src/data/run-data-update.mjs dotenv_config_path=.env",
+    "data-update-beta": "FEATURESET=beta node -r dotenv/config src/data/run-data-update.mjs dotenv_config_path=.env"
```
Comment (joeyyang) [inline package.json:38]: should this be `data-update:beta` to match the `:beta` convention elsewhere?

**Review by joeyyang — APPROVED** [review]
Looks good, just a small naming question!

**Review by lynnerang — APPROVED** [review]
Had same question about :beta naming, but otherwise looks great to me!

File: package.json (hunk 1 above)
Comment (gblakeman) [reply package.json:38]: Yeah, good call, oops. 🙃

**Pull request merged**